src/services/infinispanRefData.ts

```diff
@@ -135,3 +135,8 @@ export enum EvictionType {
   size = 'size',
   count = 'count',
 }
+
+export enum IndexedStorage {
```

Comment:
Storage type "Persistent" & "volatile" not working at last step

Reply:
@oraNod in the cache config, this storage type is not working, what should be the perfect wording for this?

Reply:
@dipguptaredhat https://docs.jboss.org/infinispan/14.0/apidocs/org/infinispan/configuration/cache/IndexStorage.html 

I believe it's

```
export enum IndexedStorage {
  persistent = 'filesystem',
  volatile = 'local_heap',
}
```